lib/flex_ui/components/carousel.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_carousel_widget/flutter_carousel_widget.dart';
+
+class FlexCarousel extends StatelessWidget {
+  const FlexCarousel({
+    super.key,
+    this.height = 270,
+    this.items,
+  });
+
+  final double height;
+  final List<Widget>? items;
+
+  @override
+  Widget build(BuildContext context) {
+    return FlutterCarousel(
+      options: CarouselOptions(
+        height: height,
+        showIndicator: true,
+        floatingIndicator: true,
+        slideIndicator: CircularSlideIndicator(
+          currentIndicatorColor: Theme.of(context).primaryColor,
```

Comment:
I would not using Material theme in our widgets, and rather declare `FlexColors.primary` explicitely.

The lack of control disturbs me here, we don't even know if `Theme.of(context).primaryColor` is the same color than `FlexColors.primary` (`FlexColors.primary` has been used as a "seed" to generate a palette, but nothing tell us if the seed becomes  the theme's primary color)